datatype/src/main/java/cz/cvut/kbss/jopa/datatype/DatatypeTransformer.java

```diff
@@ -79,6 +93,19 @@ private DatatypeTransformer() {
         return map;
     }
 
+    /**
+     * Converts the specified wrapper class to its corresponding primitive class.
+     *
+     * If the class parameter is a wrapper type, the equivalent primitive type will be returned (e.g. int.class for Integer.class)
+     * In all other cases, the return value is null.
+     *
+     * @param cls - the class to convert
+     * @return
+     */
+    public static Class<?> wrapperToPrimitive( Class<?> cls ) {
```

Comment:
1. Please don't use spaces around method parameters - let's try to use the same code style.
2. Maybe return an `Optional` instead of a `null`?
3. Consider renaming to `wrapperTypeToPrimitiveType` to be more explicit about the function's purpose.